Cargo.toml

```diff
@@ -74,6 +74,7 @@ compression-gzip = ["flate2"]
 compression-lz4 = ["lz4"]
 compression-snappy = ["snap"]
 compression-zstd = ["zstd"]
+timestamp-query = []
```

Comment:
features are mostly used to control dependencies, e.g. if you don't need compression support, then you don't need to get a fat binary with all this load. You don't add a dependency here, so no feature is required.